app/controllers/oauth_tokens_controller.rb

```diff
@@ -24,25 +24,53 @@ def refresh_token
     if json
       render json: { tokenExpiresIn: script_session_expires_in }
     else
-      render json: nil, status: 401
+      render json: nil, status: :unauthorized
     end
   end
 
-  protected
+  def store_user_data(oauth_response)
```

Comment:
Why did you remove `protected`?

Reply:
No reason, must have been removed when I removed other stuff in that file. I'll replace it.